dispatch/static/manager/src/js/actions/ArticlesActions.js

```diff
@@ -42,7 +42,12 @@ class ArticlesActions extends PublishableActions {
   }
 
   fromRemote(data) {
-    data.content = fromJSON(data.content)
+    try {
```

Comment:
This could probably be cleaner?

Reply:
What does `fromJSON(data.content)` throw if `data.content` is null? I would maybe refactor `fromJSON` to accept (and ignore) null values and then just call it normally: 

```js
function fromJSON(data) {
  if (!data) {
    return data
  }

  // convert content to JSON
}

Reply:
I like this idea